docs/implementation_progress.md

```diff
@@ -0,0 +1,145 @@
+## About
+
+When bootstrapping StableHLO from MHLO, we have inherited MHLO's implementation
+of verification. It already has significant coverage of the opset, but there's
```

Comment:
"of many things, including prettyprinting, verification and shape inference". It's not just verification that we inherited, right?